lib/sqlite-manager.js

```diff
@@ -419,8 +456,17 @@ module.exports = (function() {
   manager.addData = async function(db, data) {
```

Comment:
Can you add an `@example` usage to the JSDoc of `addData` showing how to add `ndarrays`?